README.md

```diff
@@ -18,6 +18,12 @@ OpenPBR Surface is an open standard hosted by the [Academy Software Foundation](
 * **[Reference implementation](reference/open_pbr_surface.mtlx)** – written in [MaterialX](https://materialx.org/)
 * **[BibTeX citation](openpbr.bib)**
 
+### Resources
+
+* [MaterialX Web Viewer](https://academysoftwarefoundation.github.io/MaterialX/?file=Materials/Examples/OpenPbr/open_pbr_default.mtlx) - WebGL   rasterization renderer using MaterialX [implementation](reference/open_pbr_surface.mtlx) of OpenPBR
```

Comment:
This is looking close to the mark, @portsmouth, though I still see some weird spacing in the text on this line.

Reply:
Fixed in 1ca6c3b27165a5b136758c59a808e8e106371f16